examples/flux/flux.py

```diff
@@ -0,0 +1,102 @@
+import runhouse as rh
```

Comment:
Is this meant to be in /examples or is it just a file to accompany a page in /guides? If the former, I think it needs an intro paragraph and h1 in comments about this line. Potentially a bit odd to start with an `import` statement.

Reply:
Guides + Github, not examples.